lib/pm.js

```diff
@@ -71,7 +71,15 @@ Pm.prototype.spawn = function(callback) {
 
   mkdirp(this.base);
 
-  var logFd = fs.openSync(this.logFile, 'a');
+  try {
+    fs.truncateSync(this.logFile, 0);
```

Comment:
Isn't this what the `w` flag does automatically? Is that behaviour not guaranteed on all platforms?